### ISSUE TITLE
[Feature] Ghost Mode

### ISSUE THREAD

**Issue by DaEvil1** [openlierox/openlierox#765]
Another request for race is being able to save ghosts of best laps to race against (both online and offline.) Would be a really cool feature to race against the best laps "live".

**Comment by albertz** [comment]
You mean ghosts as in records of your worms of the best race rounds?

A problem is that for each new game, the race check points are randomly recalculated.

**Comment by DaEvil1** [comment]
Yes. I don't think it's a big problem. In general the check points are very similair from round to round, so most of the race route will be very similair. It'll be a good way to improve your own racing as well as sharing good laps with the community.
